src/test/unit/test_lumigo_utils.py

```diff
@@ -454,9 +453,18 @@ def test_get_edge_host(arg, host, monkeypatch):
     assert get_edge_host("region") == host
 
 
-def test_report_json_extension_spans_mode(monkeypatch, reporter_mock):
+def test_report_json_extension_spans_mode(monkeypatch, reporter_mock, tmpdir):
+    extension_dor = tmpdir.mkdir("tmp")
+    monkeypatch.setattr(lumigo_utils, "get_extension_dir", lambda *args, **kwargs: extension_dor)
```

Comment:
more elegant way to do it is by `monkeypatch.setenv(LUMIGO_SPANS_DIR_KEY, extension_dir)`